MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillImpl.kt

```diff
@@ -118,7 +139,12 @@ internal class AddressAutofillImpl(
             )
         }
 
-        private fun List<BaseSearchResult>.toAddressAutofillSuggestions() = mapNotNull { it.toAddressAutofillSuggestion() }
+        private fun List<BaseSearchResult>.toAddressAutofillResults() = mapNotNull { it.toAddressAutofillResult() }
+
+        private fun BaseSearchResult.toAddressAutofillResult(): AddressAutofillResult? {
+            val suggestion = this.toAddressAutofillSuggestion() ?: return null
+            return AddressAutofillResult(suggestion = suggestion, address = suggestion.address)
+        }
 
         private fun BaseSearchResult.toAddressAutofillSuggestion(): AddressAutofillSuggestion? {
```

Comment:
This function should be removed, given that we've mowing away from 2 -> 1 step adapters. Once it's removed, `AddressAutofillSuggestion.underlaying` will be non-nullable property.

Reply:
Done